src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs

```diff
@@ -303,6 +306,12 @@ private int QueryFindReferencesStatus(OLECMD[] prgCmds)
             return VSConstants.S_OK;
         }
 
+        private int QuerySyncClassView(OLECMD[] prgCmds)
+        {
+            prgCmds[0].cmdf = (uint)(OLECMDF.OLECMDF_ENABLED | OLECMDF.OLECMDF_SUPPORTED);
```

Comment:
what does `command enabled` here mean? When would it not be enabled.. I imagined it was some sort of tools->options setting.